src/pages/LoginPage.tsx

```diff
@@ -36,33 +36,48 @@ const LoginPage = (): ReactElement => {
 
   return (
     <div onLoad={checkIfLoginSaved}>
-      <div className="flex flex-col justify-end min-h-[68vh] bg-gradient-to-r from-red-400 to-gray-300">
-        <div>
-          <form className="flex-col px-8 py-5 bg-transparent-gray rounded-tl-lg rounded-tr-lg shadow-xl">
-            <input
-              value={input}
-              type="text"
-              id="nuid-entry"
-              onChange={(e) => setInput(e.target.value)}
-              className="w-full bg-gray-50 border border-black text-black text-xl rounded-lg focus:ring-black-500 focus:border-black-500 block px-2.5 py-4 my-4"
-              placeholder="NUID"
-              required
-            />
-            <button
-              onClick={(e) => login()}
-              className="w-full my-2.5 bg-sga-red text-white text-2xl font-semibold rounded-lg px-2.5 py-4 hover:bg-sga-red-hover active:bg-sga-red-active"
-            >
-              Log In
-            </button>
-          </form>
-        </div>
+      <div className="flex flex-col justify-end min-h-[68vh] bg-cooper-mobile-festive md:bg-cooper-big-boy bg-cover lg:min-h-[60vh]">
+        <form className="flex-col px-8 py-5 bg-transparent-gray rounded-tl-lg rounded-tr-lg lg:invisible">
+          <input
+            value={input}
+            type="text"
+            id="nuid-entry"
+            onChange={(e) => setInput(e.target.value)}
+            className="w-full bg-gray-50 border border-black text-black text-xl rounded-lg focus:ring-black-500 focus:border-black-500 block px-2.5 py-4 my-4"
+            placeholder="NUID"
+            required
+          />
+          <button
+            onClick={(e) => login()}
+            className="w-full my-2.5 bg-sga-red text-white text-2xl font-semibold rounded-lg px-2.5 py-4 hover:bg-sga-red-hover active:bg-sga-red-active"
+          >
+            Log In
+          </button>
+        </form>
       </div>
-      <div className="flex justify-center align-center">
+      <div className="flex justify-center align-center h-64 lg:h-80">
         <img
```

Comment:
I'd recommend puttning an `object-fit: contain` here so the SGA logo doesn't get "squished" when the screen size gets smaller